scripts/publish-minor-versions.cjs

```diff
@@ -23,6 +23,8 @@ console.log('Bumping minor version to', nextVersion);
 const packagesToUpdate = [
     '@synthql/backend',
     '@synthql/cli',
+    '@synthql/handler-express',
```

Comment:
if you're going to name the packages handler-express I think you should also name the folder handler-express.